test/libyul/yulOptimizerTests/fullSuite/abi_example1.yul

```diff
@@ -465,21 +465,19 @@
 //         let _1 := 0
 //         let _2 := mload(_1)
 //         let pos := 0x20
-//         let pos_1 := pos
 //         let length := mload(_2)
 //         mstore(pos, length)
 //         pos := 64
-//         let srcPtr := add(_2, pos_1)
+//         let srcPtr := add(_2, 0x20)
```

Comment:
Why?

Reply:
This is an effect of the literal rematerializer. I couldn't tell which version is "better optimized".